bagit-pax/pitt_pax_v2.py

```diff
@@ -194,7 +222,7 @@ def fCreateOpexFragment(list_folders_in_dir, list_files_in_dir, LegacyXIP,
         desc_metadata = ET.SubElement(opex_root, opex + 'DescriptiveMetadata')
         if LegacyXIP != "":
             LegacyXIP = ET.SubElement(desc_metadata, 'LegacyXIP', {'xmlns': 'http://preservica.com/LegacyXIP'})
-            access_ref = ET.SubElement(legacyXIP, 'AccessionRef')
+            access_ref = ET.SubElement(LegacyXIP, 'AccessionRef')

```

Comment:
This drive-by correction might be better as part of its own pull request, with its own description.